packages/firestore/test/unit/remote/bloom_filter_golden_test_data/index.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { default as testDataCount1Rate1 } from './Validation_BloomFilterTest_MD5_1_1_bloom_filter_proto.json';
```

Comment:
Please rename the exports so that the test data and expected result have the same prefix. That will make it easier to correlate the two.

e.g. Instead of `testResultCount1Rate1` and `testDataCount1Rate01`, use names like `count1Rate01TestData` and `count1Rate01TestResult` or something like that.